design/FY2019/Heat Emissions from Buildings to Ambient.md

```diff
@@ -0,0 +1,216 @@
+# Heat Emissions from Buildings to Ambient
+
+Xuan Luo, Tianzhen Hong
+
+Lawrence Berkeley National Laboratory
+
+October 26, 2018 
+
+## Justification for Feature Update
+
+Heat emissions from buildings are a crucial component influencing climate change and urban microclimate, e.g., urban heat island effect. We propose to calculate heat emissions from buildings in EnergyPlus with a bottom-up approach and report them using new output variables as well as new tables in the current tabular summary reports. The heat emissions from a building include heat releases from three levels of building systems: building envelope, zone/space, and HVAC systems. The feature would allow users to quantify the dynamic behavior of heat emissions, and analyze their influencing factors including building use type, vintage, and climate. This feature directly supports DOE’s multi-year effort of enhancing EnergyPlus to support urban scale building modeling.
+
+## Overview
+
+Quantifying heat emissions from buildings and its spatiotemporal distribution have significant implications on urban environment and climate studies. Knowing how much heat is released into atmosphere can help quantify the magnitude of its impact on urban microclimate, e.g., urban heat island effect and climate change. Spatial and temporal distribution of heat emissions can serve to locate hot spots in space and time, therefore useful for prioritizing recourse to mitigate heat emissions from buildings.
+
+We propose a bottom-up approach to calculate heat emissions from buildings with any building energy modeling programs. The scope is restricted to report the buildings’ heat directly discharged to their ambient air. As illustrated in Figure 1, we propose to include heat releases from three levels of building components: (1) building envelope - exterior surface convection and radiative heat transfer to the ambient air, (2) zone - exhaust air or exfiltration to ambient, (3) HVAC system - relief/exhaust air from AHUs, and (4) HVAC system - heat rejected by air-cooled condensers or central plants, including cooling towers of water-cooled chillers, gas-fired boilers, furnaces and water heaters. It should be noted that any under-ground zone or envelope is excluded, since heat from those spaces is rejected to ground instead of to ambient air. 
+
```

Comment:
Do you include exhaust air from heat exchangers in Item 3?

Reply:
Yes. We'll include this part.